book/src/pong-tutorial/pong-tutorial-02.md

```diff
@@ -280,7 +280,7 @@ This is all the information Amethyst needs to track and move the paddles in our
 virtual world, but we'll need to do some more work to actually *draw* them.
 
 As a sanity check, let's make sure the code for initialising the paddles
-compiles. Update the `on_start` method to the following:
+successfully runs. Update the `on_start` method to the following:
```

Comment:
Shouldn't it be "runs successfully"?

Reply:
Not necessarily. In "I occasionally drink soda", the adverb is before the verb. However, "I drink soda, occasionally" is also valid. This is getting into really nit-picky grammar rules that I don't fully understand, but, while "This code runs successfully" is accepted English, I don't think it's correct by the actual rules. In any case, I don't think we should fuss about rules this deep. Language is flexible because of the demands of human nature.

Reply:
As a note, an english teacher here (I work at a college in the central USA) says that the comma in `"I drink soda, occasionally"` is incorrect and it should be `"I drink soda occasionally"`.  A comma in that position is an antiquated form that is not recommended for modern usage unless you are trying to imply an actual pause (in which case using `"I drink soda...occasionally"` is more common to show that pause; however, that is not strictly correct).